File: 2022-2023/sourcecode/lab/lab13/OfflineCachingBasicsJava/app/src/main/java/pl/udu/uwr/pum/offlinecachingbasicsjava/data/remote/RandomApi.java
package pl.udu.uwr.pum.offlinecachingbasicsjava.data.remote;

import java.util.List;

import pl.udu.uwr.pum.offlinecachingbasicsjava.data.User;
import retrofit2.Call;
import retrofit2.http.GET;

public interface RandomApi {
    @GET("users?size=20")
    Call<List<User>> users();
}

File: 2022-2023/sourcecode/lab/lab13/OfflineCachingBasicsKotlin/app/src/main/java/pl/udu/uwr/pum/offlinecachingbasicskotlin/data/remote/RandomApi.kt
package pl.udu.uwr.pum.offlinecachingbasicskotlin.data.remote

import pl.udu.uwr.pum.offlinecachingbasicskotlin.data.User
import retrofit2.Response
import retrofit2.http.GET

interface RandomApi {
    @GET("users?size=20")
    suspend fun users(): List<User>
}